src/null-safety/faq.md

```diff
@@ -0,0 +1,292 @@
+---
+title: Null safety frequently asked questions
+description: FAQs for Dart's null safety migration
```

Comment:
This is the page's social sharing text, so let's make it friendly and "you" focused.

```suggestion
description: FAQs to help you migrate your Dart code to null safety
```